source/reference/known-issues.txt

```diff
@@ -68,3 +68,26 @@ Given the exact same |sql| query, the |bi-short| does not return
       GROUP BY class;
       
       Empty set (0.00 sec)
+
+
+BI Connector Returns ``NULL`` Values Due to Mapping Error
+----------------------------------------------------------
+
+A known mapping error can occur that causes |bi-short| to return all ``NULL`` 
+values. This error occurs when you query a collection that
+contains an array field that can be ``NULL``, and the same array contains 
+an additional field that can also be ``NULL``. A workaround for this error is to 
+create a view by including the ``nonnullsched`` parameter with the 
```

Comment:
nonnullsched is the name of the view in this case. The important thing is the match that filters on null :)